pagopa/pn-frontend
# Feat/pn 4164 pn 4279 disclaimer modal contacts and io sections

## Short description
This PR adds the DisclaimerModal component in pn-commons. 
Used in pf and pg webapps in the contacts sections to alert and inform the users that logging in and viewing the notification within 5 days by the email reception will not receive the physical notification.
Used in the IO servicesection to alert the user about the activation in the IO app.

## List of changes proposed in this pull request
- Adds DisclaimerModal in pn-commons
- Adds tests for DisclaimerModal
- Adds alert before IO service activation and deactivation
- Adds alert before adding or editing a non-PEC email or phone contact
- Fixes unit tests for personafisica and personagiuridica webapps
- Fixes e2e tests for personafisica and personagiuridica webapps


## How to test
To test app IO service activation change the env to COLL (user: matteo)
- Try to enable the IO service, a modal with a checkbox should appear
- Try to disable the IO service, a modal without a checkbox should appear
- Try to add a phone or email address in courtesy and special contacts, a modal with a checkbox should appear before the verification code modal
